.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -181,30 +182,41 @@ jobs:
         THREADS_LIMIT: 20
         RUN_TYPE: 'perf_ci'
         ENABLE_PROMETHEUS_SNAPSHOT: 'True'
-        WINDOWS_URL: ${{ secrets.PERF_WINDOWS_URL }}
+        WINDOWS_SERVER_2019_URL: ${{ secrets.PERF_WINDOWS_SERVER_2019_URL }}
+        WINDOWS11_URL: ${{ secrets.PERF_WINDOWS11_URL }}
       run: |
         build=$(pip freeze | grep benchmark-runner | sed 's/==/=/g')
         build_version="$(cut -d'=' -f2 <<<"$build")"
         echo '>>>>>>>>>>>>>>>>>>>>>>>>>> Start E2E workload: ${{ matrix.workload }} >>>>>>>>>>>>>>>>>>>>>>>>>>'
         scp -r "$RUNNER_PATH/.kube/config" provision:"$CONTAINER_KUBECONFIG_PATH"
-        # SCALE: Run vdbench/ bootstorm
-        if [[ '${{ matrix.workload }}' == 'vdbench_pod_scale' || '${{ matrix.workload }}' == 'vdbench_kata_scale' || '${{ matrix.workload }}' == 'vdbench_vm_scale' || '${{ matrix.workload }}' == 'bootstorm_vm_scale' || '${{ matrix.workload }}' == 'windows_vm_scale' ]]
+        workload=$(awk -F_ '{print $1"_"$2}' <<< '${{ matrix.workload }}')
+        run=$(awk -F_ '{print $3}' <<< 'windows_vm_scale_windows_server_2019')
+        # SCALE RUN
+        if [[ '$run' == 'scale' ]]
         then
-            workload=$(awk -F_ '{print $1"_"$2}' <<< '${{ matrix.workload }}')
             # bootstorm_vm_scale: no need redis for synchronization but need SCALE and THREADS_LIMIT
             if [[ '${{ matrix.workload }}' == 'bootstorm_vm_scale' ]]
             then
               # Warm-up: Pull the Fedora image from quay.io for each node
               ssh -t provision "podman run --rm -t -e WORKLOAD='$workload' -e KUBEADMIN_PASSWORD='$KUBEADMIN_PASSWORD' -e SCALE='$SCALE' -e SCALE_NODES=$SCALE_NODES -e REDIS='$REDIS' -e RUN_ARTIFACTS_URL='$RUN_ARTIFACTS_URL' -e BUILD_VERSION='$build_version' -e RUN_TYPE='$RUN_TYPE' -e KATA_CPUOFFLINE_WORKAROUND='True' -e SAVE_ARTIFACTS_LOCAL='False' -e ENABLE_PROMETHEUS_SNAPSHOT='$ENABLE_PROMETHEUS_SNAPSHOT' -e THREADS_LIMIT='$THREADS_LIMIT' -e WINDOWS_URL='$WINDOWS_URL' -e TIMEOUT='$TIMEOUT' -e log_level='INFO' -v '$CONTAINER_KUBECONFIG_PATH':'$CONTAINER_KUBECONFIG_PATH' --privileged 'quay.io/ebattat/benchmark-runner:latest'"
               SCALE=$BOOTSTORM_SCALE
-            elif [[ '${{ matrix.workload }}' == 'windows_vm_scale' ]]
+            elif [[ '$workload' == 'windows_vm' ]]
             then
+              if [[ '${{ matrix.workload }}' == 'windows_vm_scale_windows_server_2019' ]]
+              then
+                WINDOWS_URL=$WINDOWS_SERVER_2019_URL
+              fi
+              if [[ '${{ matrix.workload }}' == 'windows_vm_scale_windows11' ]]
+              then
+                WINDOWS_URL=$WINDOWS11_URL
+              fi
```

Comment:
Should be `elif` rather than two separate if clauses.

Reply:
I prefer to keep it like this if we plan to add other windows os, as windows10

Reply:
You can chain elif clauses, but this is really the right way to do it:

```
case '${{ matrix_workload }}' in
    windows_vm_scale_windows_server_2019) WINDOWS_URL="$WINDOWS_SERVER_2019_URL" ;;
    windows_vm_scale_windows11) WINDOWS_URL="$WINDOWS11_URL" ;;
    *) echo "Unknown Windows scale workload ${{ matrix_workload }}"; exit 1 ;;
esac
```

Then it's much easier to add other variants.

Reply:
@RobertKrawitz, fix it to if-else approach, not required case for simple condition

Reply:
This situation is exactly what case/esac is intended for.

Reply:
Fixed